manim/mobject/types/vectorized_mobject.py

```diff
@@ -1014,16 +1037,68 @@ def add(self, *vmobjects):
 
         Returns
         -------
-        None
+        VGroup
 
         Raises
         ------
         TypeError
             If one element of the list is not an instance of VMobject
+
+        Examples
+        --------
+        .. manim:: AddToVGroup
+
+            class AddToVGroup(Scene):
+                def construct(self):
+                    circle_red = Circle(color=RED)
+                    circle_green = Circle(color=GREEN)
+                    circle_blue = Circle(color=BLUE)
+                    circle_red.shift(LEFT)
+                    circle_blue.shift(RIGHT)
+                    gr = VGroup(circle_red, circle_green)
+                    gr2 = VGroup(circle_blue) # Constructor uses add directly
+                    self.add(gr,gr2)
+                    self.wait()
+                    gr += gr2 # Add group to another
+                    self.play(
+                        gr.shift, DOWN,
+                    )
+                    gr -= gr2 # Remove group
+                    self.play( # Animate groups separately
+                        gr.shift, LEFT,
+                        gr2.shift, UP,
+                    )
+                    self.play( #Animate groups without modification
+                        (gr+gr2).shift, RIGHT
+                    )
+                    self.play( # Animate group without component
+                        (gr-circle_red).shift, RIGHT
+                    )
         """
         if not all(isinstance(m, VMobject) for m in vmobjects):
             raise TypeError("All submobjects must be of type VMobject")
-        super().add(*vmobjects)
+        return super().add(*vmobjects)
+
+    def __add__(self, vmobject):
+        copy = VGroup()
+        for submobject in self.submobjects:
+            copy.add(submobject)
+        copy.add(vmobject)
+        return copy
+
+    def __iadd__(self, vmobject):
+        return self.add(vmobject)
+
+    def __sub__(self, vmobject):
+        copy = VGroup()
+        for submobject in self.submobjects:
+            if submobject == vmobject:
+                continue
+            copy.add(submobject)
+        return copy
```

Comment:
I would rather do the following:
```suggestion
    def __sub__(self, vmobject):
        copy = VGroup(*self.submobjects)
        copy.remove(vmobject)
        return copy
```
This calls fewer methods (one constructor and call to `remove` instead of one call to `add` for each mobject). But also because the original code uses the comparison operator `==`. This operator could be overriden in the future (for example, if you have two mobjects of the same class, position, and color, should they be the same?). Instead, the call to `remove` uses the `in` operator of a list, which [is more robust](https://stackoverflow.com/a/48861469/14157230).